contractsV5/FeeHandler.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity 0.5.11;
+
+import "./IKyberDAO.sol";
+import "./KyberNetwork.sol";
+import "./IFeeHandler.sol";
+import "./PermissionGroupsV5.sol";
+import "./UtilsV5.sol";
+
+contract FeeHandler is IFeeHandler, Utils {
+
+    IKyberDAO public kyberDAOContract;
+    KyberNetwork public kyberNetworkContract;
+
+    // Todo: Add the correct startBlock and epoch duration values
+    uint constant STARTBLOCK = 0;
```

Comment:
we will not save those values.

the expiry block parameter saves the need to save locally.
so only DAO knows about epoch number and duration.
so can remove these values

Reply:
Mike mentioned in the group (https://t.me/c/1407851547/3537) that I can hardcode `STARTBLOCK` and `EPOCHDURATION` in the FeeHandler. I can remove it if it is not needed. I don't think I need it.

Reply:
yes.
but later I asked him to decouple the logic.

its much better if the epoch logic is only in one place
the more we decouple logic - its better design

if later epoch logic changes.
fee handler doesn't have to be modified.

Reply:
noted